waspc/data/Generator/templates/server/src/auth/providers/email/resetPassword.ts

```diff
@@ -0,0 +1,19 @@
+import { handleRejection } from "../../../utils.js";
+import { findUserBy, updateUserPassword, verifyToken } from "../../utils.js";
+
+export const resetPassword = handleRejection(async (req, res) => {
+    const args = req.body || {};
+    const { token, newPassword } = args;
+    try {
+        const { id: userId } = await verifyToken(token);
```

Comment:
How long are tokens valid for? Should we make it shorter than our default expiration?

Reply:
I've updated the code to make the tokens valid for 30 minutes.

Reply:
TODO: make sure we redirect to a user defined route when the token is invalid or expired